Tests/Functional/ContinueAfterFailureTestCase/main.swift

```diff
@@ -0,0 +1,41 @@
+// RUN: %{swiftc} %s -o %T/ContinueAfterFailureTestCase
+// RUN: %T/ContinueAfterFailureTestCase > %t || true
+// RUN: %{xctest_checker} %t %s
+
+#if os(macOS)
+    import SwiftXCTest
+#else
+    import XCTest
+#endif
+
+// CHECK: Test Suite 'All tests' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+// CHECK: Test Suite '.*\.xctest' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+
+// CHECK: Test Suite 'ContinueAfterFailureTestCase' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+class ContinueAfterFailureTestCase: XCTestCase {
+    static var allTests = {
+        return [
+            ("testDoesNotContinueAfterFailure", testDoesNotContinueAfterFailure),
+            ("testContinueAfterFailure", testContinueAfterFailure)
+        ]
+    }()
+    
+    // CHECK: Test Case 'ContinueAfterFailureTestCase.testDoesNotContinueAfterFailure' started at \d+-\d+-\d+ \d+:\d+:\d+\.\d+
+    // CHECK: First Log in DoesNotContinueAfterFailure test
+    // CHECK: .*/ContinueAfterFailureTestCase/main.swift:[[@LINE+4]]: error: ContinueAfterFailureTestCase.testDoesNotContinueAfterFailure : XCTAssertTrue failed -
```

Comment:
You should include a `CHECK-NOT` for "Second log" here, to verify that it's not being printed.

Reply:
Ok I will add it! I'm not very familiar with lit